woocommerce/woocommerce-ios
# Add unit tests to FeeLineDetailsViewModelTests

### Description
This PR adds two assertions to test items that were not yet tested.
**First:** `.isExistingFeeLine` already has an `AssertTrue` in `test_view_model_prefills_input_data_correctly`, but an `AssertFalse` can alert us to a potential situation where initialization of `.isExistingFeeLine` is broken.
**Second:** `amountPlaceholder` was not being tested yet. It should always be `"0"`, so I used AssertEqual to ensure we do not pass an empty string instead of `"0"`.

### Testing instructions
Both asserts should pass if you run `test_view_model_formats_amount_correctly`.
Then, try to make them fail:
1. Try initializing with `.isExistingFeeLine` set to true. (line 116 of `FeeLineDetailsViewModel`)
2. Try passing an empty string, or any string other than "0". (line 114 of `FeeLineDetailsViewModel`)

Can you think of other ways to break these? Especially `.isExistingFeeLine`?



One thing I am wondering:
I placed these within `test_view_model_formats_amount_correctly`, as it uses a blank initial viewModel. However, these asserts do not have to do with formatting the amount.
Would it be best to: create a new test, rename the existing test, or not worry about it since the asserts will make it clear what part failed?

---
